rest/adminapitest/admin_api_test.go

```diff
@@ -3033,7 +3034,17 @@ func TestDbOfflineConfigPersistent(t *testing.T) {
 	// Check offline config matches online config
 	resp = rt.SendAdminRequest(http.MethodGet, "/{{.db}}/_config", "")
 	rest.RequireStatus(t, resp, http.StatusOK)
```

Comment:
This test case is actually testing what used to happen and we should replace `rt.TakeDbOffline()` with something that _doesn't_ modify the stored config for this test instead of adding code to accommodate the changed fields.

We're ensuring that the database is able to restart and come back online, or that a single node can be taken offline without affecting others.